doctr/models/detection/differentiable_binarization/pytorch.py

```diff
@@ -208,11 +208,16 @@ def forward(
         if return_model_output:
             out["out_map"] = prob_map
 
+        def need_conversion_to_float(dtype):
+            # pytorch: torch/csrc/utils/tensor_numpy.cpp:aten_to_numpy_dtype
+            return dtype in [torch.bfloat16]
+
+        numpy_dtype_converter = lambda x: x.float() if need_conversion_to_float(x.dtype) else x
```

Comment:
directly checking `dtype in [torch.bfloat16]` is simpler?

Reply:
Updated as suggested.